catalog/installWebSocket.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# use the command line interface to install Weather.com package.
+#
+: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH must be set and non-empty"}
+AUTH_KEY=$WHISK_SYSTEM_AUTH
+
+SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
+CATALOG_HOME=$SCRIPTDIR
+source "$CATALOG_HOME/util.sh"
+
+echo Installing WebSocket package.
+
+createPackage websocket \
+    -a description "Utilities for communicating with WebSockets"
+
+waitForAll
+
```

Comment:
re: separability concerns, yes. we can do 1-2, and the 3 as two separate tasks, but we have to make sure that we don't keep kicking the 3 can down the road.